### PR TITLE
Recipe App

### PR DESCRIPTION
**Pull request by AbbasSarwar**
### Features
- **[Recipe add]**
- **[Ingredients add]**
- **[Shopping list]**
- **[Public enable]**
- **[Kanban board]**

* Integration Tests
* Unit Tests

### CODE REVIEW

File: app/controllers/ingredients_controller.rb [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+      end
+    end
+  end
+
+  def edit
+    @ingredient = current_user.ingredients.find(params[:id])
+    @recipe = current_user.recipe
+  end
+
+  def update
```
Comment (joskalenda) [inline app/controllers/ingredients_controller.rb:36]: - [ ] We are missing the update method on the front end as per the wireframe, the user needs to be able to modify the ingredient, see? please fix this
- [ ] YOU
- [ ] 


- [ ] EXPECTED

File: app/controllers/ingredients_controller.rb [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+      if @ingredient.update(ingredient_params)
+        format.html { redirect_to ingredient_url(@ingredient), notice: 'Ingredient was successfully updated.' }
+        format.json { render :show, status: :ok, location: @ingredient }
+      else
+        format.html { render :edit, status: :unprocessable_entity }
+        format.json { render json: @ingredient.errors, status: :unprocessable_entity }
+      end
+    end
+  end
+
```
Comment (joskalenda) [inline app/controllers/ingredients_controller.rb:48]: - [ ] Please take a closer look at this wireframe and make the necesary update to your design, add ingredient button should be on the right and shopping list on the left, and make sure to widen your table to fit the screen for UI
EXPECTED



YOURS

**Review by joskalenda — CHANGES_REQUESTED** [review]
| STATUS: REQUIRED CHANGES ♻️  |
|--------------------------------|
| Hi @AbbasSarwar , |
|Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there! |
|  |

## Highlights

- [ ].. ✅ Readme
- [ ].. ✅ Clean code

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

-------
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@joskalenda) in your question so I can receive the notification.

> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/controllers/ingredients_controller.rb (hunk 1 above)
Comment (AbbasSarwar) [reply app/controllers/ingredients_controller.rb:36]: Done as advised.

File: app/controllers/ingredients_controller.rb (hunk 2 above)
Comment (AbbasSarwar) [reply app/controllers/ingredients_controller.rb:48]: Done as advised.

**Comment by tugsem** [comment]
## STATUS: APPROVED 👍🎯  
Hello 👋 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights ⭐ 
✔️update button added to the ingredients
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**